AdminRouter/start-windows-build.ps1

```diff
@@ -0,0 +1,266 @@
+}
+
+function Get-RemoteBuildDirectoryPath {
+    $relativePath = Get-AdminRouterBuildRelativePath
+    return "$REMOTE_ADMINROUTER_BUILD_DIR/$relativePath"
+}
+
+function Get-BuildOutputsUrl {
+    $relativePath = Get-AdminRouterBuildRelativePath
+    return "$ADMINROUTER_BUILD_BASE_URL/$relativePath"
+}
+
+function New-RemoteLatestSymlinks {
+    $remoteDirPath = Get-RemoteBuildDirectoryPath
+    $baseDir = (Split-Path -Path $remoteDirPath -Parent) -replace '\\', '/'
+    New-RemoteSymlink -RemotePath $remoteDirPath -RemoteSymlinkPath "$baseDir/latest"
+    $repoDir = (Split-Path -Path $baseDir -Parent) -replace '\\', '/'
+    New-RemoteSymlink -RemotePath $remoteDirPath -RemoteSymlinkPath "$repoDir/latest"
+}
+
+function Start-LogServerFilesUpload {
+    $consoleUrl = "${JENKINS_SERVER_URL}/job/${env:JOB_NAME}/${env:BUILD_NUMBER}/consoleText"
+    Start-FileDownload -Force -URL $consoleUrl -Destination "$ADMINROUTER_BUILD_LOGS_DIR\console-jenkins.log"
+    $remoteDirPath = Get-RemoteBuildDirectoryPath
+    New-RemoteDirectory -RemoteDirectoryPath $remoteDirPath
+    Copy-FilesToRemoteServer "$ADMINROUTER_BUILD_OUT_DIR\*" $remoteDirPath
+    $buildOutputsUrl = Get-BuildOutputsUrl
+    $global:PARAMETERS["BUILD_OUTPUTS_URL"] = $buildOutputsUrl
+    Write-Output "Build artifacts can be found at: $buildOutputsUrl"
+    if($global:PARAMETERS["BUILD_STATUS"] -eq "PASS") {
+        New-RemoteLatestSymlinks
+    }
+ }
+
+function Start-EnvironmentCleanup {
+    # Stop any potential hanging process
+    $processes = @('go', 'bash')
+    $processes | Foreach-Object { Stop-Process -Name $_ -Force -ErrorAction SilentlyContinue }
+    cmd.exe /C "rmdir /s /q $ADMINROUTER_DIR > nul 2>&1"
+}
+
+function Get-SuccessBuildMessage {
+    return "Successful DC/OS AdminRouter Windows build and testing for repository $GitURL on $Branch branch"
+}
+
+function Start-TempDirCleanup {
+    Get-ChildItem $env:TEMP | Where-Object {
+        $_.Name -notmatch "^jna\-[0-9]*$|^hsperfdata.*_diagnostics$"
+    } | ForEach-Object {
+        $fullPath = $_.FullName
+        if($_.FullName -is [System.IO.DirectoryInfo]) {
+            cmd.exe /C "rmdir /s /q ${fullPath} > nul 2>&1"
+        } else {
+            cmd.exe /C "del /Q /S /F ${fullPath} > nul 2>&1"
+        }
+    }
+}
+
+function New-ParametersFile {
+    Param(
+        [Parameter(Mandatory=$true)]
+        [string]$FilePath
+    )
+    if(Test-Path $FilePath) {
+        Remove-Item -Force $FilePath
+    }
+    New-Item -ItemType File -Path $FilePath | Out-Null
+}
+
+function Write-ParametersFile {
+    Param(
+        [Parameter(Mandatory=$true)]
+        [string]$FilePath
+    )
+    if($global:PARAMETERS["LOGS_URLS"]) {
+        $global:PARAMETERS["LOGS_URLS"] = $global:PARAMETERS["LOGS_URLS"] -join '|'
+    }
+    $json = ConvertTo-Json -InputObject $global:PARAMETERS
+    Set-Content -Path $FilePath -Value $json
+}
+
+try {
+    $ErrorActionPreference = 'SilentlyContinue'
```

Comment:
Why this `$ErrorActionPreference = 'SilentlyContinue'` ?

Reply:
No need to have that line, I will remove it.